README.md

```diff
@@ -94,6 +104,17 @@ This will plan a deployment, but not execute it -- useful to see ahead what chan
 
 The big kahuna! This will deploy all of your changes, including redeploying lambdas if any of code changes.
 
+## Query
+
+#### `./indexer query <CID>`
+Attempts to find the given CID in the Indexer node. The result is a Location Claim that needs to be used to fetch the actual content associated with that CID. In case you want to query a specific node, you can use the following command:
```

Comment:
strictly speaking, the result can be several location claims (there may be several storage nodes that store a copy of the content) for both the content and potential indexes, as well as the indexes themselves.